From: Gerd Hoffmann <kraxel@redhat.com>
Date: Thu, 6 May 2010 10:50:41 +0200
Subject: Add qemu qxl vga pci bios

Add PCI vgabios for the qemu qxl vga (1b36:0100).
Name it vgabios-qxl.bin.

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
---
 Makefile |   10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

--- a/Makefile
+++ b/Makefile
@@ -14,7 +14,7 @@
 
 VGABIOS_DATE = "-DVGABIOS_DATE=\"$(RELDATE)\""
 
-all: bios cirrus-bios hires-bios vmware-bios
+all: bios cirrus-bios hires-bios vmware-bios qxl-bios
 
 bios: vgabios.bin vgabios.debug.bin
 
@@ -24,6 +24,8 @@
 
 vmware-bios: vgabios-vmware.bin vgabios-vmware.debug.bin
 
+qxl-bios: vgabios-qxl.bin vgabios-qxl.debug.bin
+
 clean:
 	/bin/rm -f  biossums vbetables-gen vbetables.h *.o *.s *.ld86 \
           temp.awk.* vgabios*.orig _vgabios_* _vgabios-debug_* core vgabios*.bin vgabios*.txt $(RELEASE).bin *.bak
@@ -43,6 +45,8 @@
 vgabios-hires.debug.bin  : VGAFLAGS := -DVBE -DPCIBIOS -DHIRES -DDEBUG
 vgabios-vmware.bin       : VGAFLAGS := -DVBE -DPCIBIOS -DPCI_VID=0x15ad -DPCI_DID=0x0405
 vgabios-vmware.debug.bin : VGAFLAGS := -DVBE -DPCIBIOS -DPCI_VID=0x15ad -DPCI_DID=0x0405 -DDEBUG
+vgabios-qxl.bin          : VGAFLAGS := -DVBE -DPCIBIOS -DPCI_VID=0x1b36 -DPCI_DID=0x0100
+vgabios-qxl.debug.bin    : VGAFLAGS := -DVBE -DPCIBIOS -DPCI_VID=0x1b36 -DPCI_DID=0x0100 -DDEBUG
 
 # dist names
 vgabios.bin              : DISTNAME := VGABIOS-lgpl-latest.bin
@@ -53,6 +57,8 @@
 vgabios-hires.debug.bin  : DISTNAME := VGABIOS-lgpl-latest.hires.debug.bin
 vgabios-vmware.bin       : DISTNAME := VGABIOS-lgpl-latest.vmware.bin
 vgabios-vmware.debug.bin : DISTNAME := VGABIOS-lgpl-latest.vmware.debug.bin
+vgabios-qxl.bin          : DISTNAME := VGABIOS-lgpl-latest.qxl.bin
+vgabios-qxl.debug.bin    : DISTNAME := VGABIOS-lgpl-latest.qxl.debug.bin
 
 # dependencies
 vgabios.bin              : $(VGA_FILES) $(VBE_FILES) biossums
@@ -63,6 +69,8 @@
 vgabios-hires.debug.bin  : $(VGA_FILES) $(VBE_FILES) biossums
 vgabios-vmware.bin       : $(VGA_FILES) $(VBE_FILES) biossums
 vgabios-vmware.debug.bin : $(VGA_FILES) $(VBE_FILES) biossums
+vgabios-qxl.bin          : $(VGA_FILES) $(VBE_FILES) biossums
+vgabios-qxl.debug.bin    : $(VGA_FILES) $(VBE_FILES) biossums
 
 # build rule
 %.bin:
